testing/simulator/src/cli.rs

```diff
@@ -1,12 +1,12 @@
-use clap::{App, Arg, SubCommand};
+use clap::{Arg, ArgAction, Command};
 
-pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
-    App::new("simulator")
-        .version(crate_version!())
+pub fn cli_app() -> Command {
+    Command::new("simulator")
+        // .version(crate_version!())
```

Comment:
can't do version anymore?

Reply:
It's possible, you need to add the feature
```yaml
clap = { version = "4.5.1", features = ["cargo"] }
```
and import
```rust
use clap::{crate_version, Arg, ArgAction, Command};
```